.github/workflows/hosted_runners.yml

```diff
@@ -537,7 +537,7 @@ jobs:
       matrix:
         build_type: [Release, Debug]
         architecture: [x86_64, arm64]
-        os: [macos-14]
+        os: [macos-14, macos-15]
```

Comment:
I think doubling isn't wrong. We often want an early one, and bringing in a recent one for test coverage seems important.